cmd/prom-aggregation-gateway/main.go

```diff
@@ -259,12 +301,15 @@ func (a *aggate) handler(w http.ResponseWriter, r *http.Request) {
 }
 
 func main() {
-	listen := flag.String("listen", ":80", "Address and port to listen on.")
+	listen := flag.String("listen", ":8080", "Address and port to listen on.")
```

Comment:
This would be a breaking change for users, and unrelated to the description.  Please remove from this PR.